components/ProjectsView.vue

```diff
@@ -42,28 +42,37 @@ function deleteProject(slug) {
 
 <template>
   <div class="px-4">
-    <ul class="project-list" role="list">
-      <li v-for="project in projects" :key="project.slug">
-        <div class="flex-none w-32">
-          <p class="project-name">
-            {{ project.name }}
-          </p>
-          <p class="project-type">
-            Type: {{ project.type }}
-          </p>
-        </div>
-        <ProjectButtons :project="project" />
-        <div class="w-64">
-          <span class="text-sm font-medium">Project usage: <b>{{ project.usage }} </b></span>
+    <div class="project-list" role="list">
+      <div
+        v-for="project in projects" :key="project.slug"
+        class="flex flex-col sm:flex-row items-center gap-1 py-4 my-3 border"
```

Comment:
You can add consistent padding all the way around here (E.g. `p-2`) so that the buttons don't butt up against the side on mobile. You can change/disable this padding on larger views. Then get rid of the `ml-4` on the two divs inside and justify the trashcan to the right instead of center. I'd also get rid of `items-center` so the button is top-aligned instead of visually floating.

Here's what it looks like currently:
![image](https://github.com/planship/planship-nuxt-example/assets/827891/47c20ca2-77ea-497f-b1c1-7d845e8f924c)

And here's what it would look like with this suggestion:
![image](https://github.com/planship/planship-nuxt-example/assets/827891/f8fb62b5-7af0-4c40-bbe2-c5225146a7b5)